generator-input/pipeline-config.json

```diff
@@ -48,6 +48,34 @@
           "defaultValue": "https://github.com/googleapis/google-cloud-dotnet"
         }
       ]
+    },
+    "publish-library": {
+      "environmentVariables": [
+        {
+          "name": "NUGET_API_KEY_GOOGLE_APIS_PACKAGES",
+          "secretName": "nuget-api-key-google-apis-packages"
+        },
+        {
+          "name": "NUGET_API_KEY_GOOGLE_CLOUD",
+          "secretName": "nuget-api-key-google-cloud"
+        },
+        {
+          "name": "DOCS_BUCKET_GOOGLEAPISDEV",
+          "secretName": "docs-bucket-googleapisdev"
+        },
+        {
+          "name": "DOCS_CREDENTIALS_GOOGLEAPISDEV",
+          "defaultValue": "adc"
```

Comment:
We need to give permissions to the 1PP account so that for this. I'll show Laura.

Also, we could maybe drop these env vars entirely, wdyt?

Reply:
Done in a third commit.